sorc/ncep_post.fd/MISCLN.f

```diff
@@ -3252,7 +3283,9 @@ SUBROUTINE MISCLN
            DPBND = 300.E2
            CALL CALCAPE(ITYPE,DPBND,P1D,T1D,Q1D,LB2,EGRID1,     &
                         EGRID2,EGRID3,EGRID4,EGRID5)
-           IF (SUBMODELNAME == 'RTMA') MUMIXR(I,J) = Q1D(I,J)
+!           IF (SUBMODELNAME == 'RTMA' .OR. MODELNAME == 'FV3R') THEN
```

Comment:
May I ask the reason of adding model name "FV3R"?

Reply:
This additional conditional model check (FV3R) was added in order to ensure that the most unstable mixing ratio field would be accessible as an output product for the RRFS-based 3D-RTMA workflow.